kafka/kafka-impl/src/main/java/com/datastax/oss/sink/kafka/CassandraSinkConnector.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.sink.kafka;
+
+import com.datastax.oss.sink.config.CassandraSinkConfig;
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.Task;
+import org.apache.kafka.connect.sink.SinkConnector;
+
+/** Sink connector to insert Kafka records into Apache Cassandra or DSE. */
+public class CassandraSinkConnector extends SinkConnector {
+
+  public static final String KAFKA_CONNECTOR_APPLICATION_NAME = "DataStax Apache Kafka Connector";
+
+  private String version;
+  private Map<String, String> properties;
+
+  @Override
+  public String version() {
```

Comment:
I suspect that the version method could be reused between this class and `PulsarSinkConnector`.
We could pass the path to version as a parameter here. Maybe we can extract it to a `SinkConnectorBase` (or similar) in the `commons` package?

Reply:
I don't think so. while kafka is stable to some extent and rarely changes, pulsar sink may change frequently as it's young. so, they're going to be of different versions

Reply:
yes, the version is in the version file. The version file path could be different and passed as an argument from kafka/pulsar The rest of the logic is the same.